integration/resources/compose/boulder.yml

```diff
@@ -0,0 +1,44 @@
+boulder:
+    image: trecloux/boulder:20161209-8a6382f4
```

Comment:
No change, I just built/pushed the last commit of the release branch 8a6382f4 is the commit id. The official image was not updated for 9 months.